examples/cmd/snapshotting/README.md

```diff
@@ -62,20 +62,5 @@ Pressing enter resumes execution of the program.
 You can remove dependencies via a simple `make clean`.
 
 ```
-sudo make clean
-```
-
-## Issues
-
-You may encounter an issue where the image does not build properly. This is often indicated via the following near the end of terminal output:
-
-```
-umount: /firecracker/build/rootfs/mnt: not mounted.
-```
-
-This is due to an issue in Firecracker's devtool command used to dynamically create an image. Fixing this is often as simple as rerunning the command.
-
-```
-sudo rm -rf root-drive-with-ssh.img root-drive-ssh-key
-sudo make image
+make clean
```

Comment:
what was changed here that we no longer need sudo?

Reply:
Devtool used to build rootfs uses docker --privileged, then generated rootfs file are under root, needs sudo to copy to test location. 
The new approach directly download from S3 no longer needs that.